### PR TITLE
Statistic stuff

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: templates/content/stats.html.twig [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+    <div class="chart chart-month">
+        {% if stats %}
+            {% for key, value in stats %}
+                <div class="graph-column">
+                    <div class="graph-counterValue">{{ value }}</div>
+                    <div class="graph-bar" style="height: {{ (value / max(stats)) * 300 }}px;"></div>
+                    <div class="graph-legend-x">{{ key|capitalize }}</div>
+                </div>
+            {% endfor %}
+        {% else  %}
```
Comment (lyrixx) [inline templates/content/stats.html.twig]: You could use for / else feature

Comment (pyrech) [reply templates/content/stats.html.twig]: Indeed, I wanted to do that before but forgot and just copy/paste this block :smile:

Comment (pyrech) [reply templates/content/stats.html.twig]: (done)

**Comment by damienalexandre** [comment]
We will lose the statistics already in place so when deploying, someone could create the new counters with the initial values :+1:

**Comment by pyrech** [comment]
Yep, that's planned :wink:

**Pull request merged**